packages/bridge-ui/src/app.css

```diff
@@ -35,6 +35,16 @@
   border-color: hsla(var(--a) / var(--tw-bg-opacity, 1));
 }
 
+.btn.btn-accent.approve-btn {
+  background-color: #4c1d95;
+  border-color: #4c1d95;
+}
+
+.btn.btn-accent.approve-btn:hover {
```

Comment:
Why not to add this specific styling into the component?, feels like it should be scoped

Reply:
Made the change, thanks for pointing it out.